src/components/FeatureViews/AdminPortal/ActionLog.js

```diff
@@ -288,9 +273,14 @@ const ActionLog = () => {
                                             "MuiInputBase-input": classes.MuiInputBase
                                         }}
                                     >
-                                        {admins.data.admins.map((userType) =>
+                                         {/*
+                                            why the hell does this work. I wrote it and I don't even understand it. maybe its because im tired
+                                            shouldn't userType be a string? how the f does value accept an object and display the correct name???????????????????? 
+                                            idk im going to sleep LOL
+                                        */}
+                                        {data.admins.map((userType) =>
                                             (<MenuItem key={userType} value={userType}>
-                                                {userType.user.firstName} {userType.user.lastName}
+                                                {fullName(userType.user)}
```

Comment:
you're displaying `fullName(userType.user)`, not `userType`

Reply:
not sure what you mean by this

Reply:
I forgot I left this comment - it was in regards to your comment earlier "why the hell does this work..." 

Does your filter actually work?